src/pages/home/HeaderView.js

```diff
@@ -19,6 +19,10 @@ const propTypes = {
     // Decides whether we should show the navigationMenu button
     shouldShowNavigationMenuButton: PropTypes.bool.isRequired,
 
+    // Report ID currently being looked at, use to retrieve more information about the report.
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportID: PropTypes.string.isRequired,
```

Comment:
Is there a better way to do this? I passed in the reportID from the `HomePage` since I wasn't sure how else we'd know what report to pull from Onyx...

Reply:
Ah hmm yeah this might be a limitation of Onyx since the alternative would be to get the `currentlyViewedReportID` from Onyx, but Onyx also needs this to get the report? Is that what you are asking?

Reply:
This seems fine.

Reply:
Ah hmm yeah this might be a limitation of Onyx since the alternative would be to get the currentlyViewedReportID from Onyx, but Onyx also needs this to get the report? Is that what you are asking?

Yes, precisely.

Reply:
Ok awesome. I think this is the best we can do for now. But interesting thing we haven't run into before. I'm fine with this.